e2e/cypress/support/api/cloud_default_config.json

```diff
@@ -260,7 +260,7 @@
         }
     },
     "DisplaySettings": {
-        "CustomUrlSchemes": [],
+        "CustomURLchemes": [],
```

Comment:
There's a typo here that's hopefully not affecting too many files

Reply:
Fixed it.

Reply:
Thanks for catching the typo. Actually, `CustomUrlSchemes` is correct as it comes from the client manifest.

Reply:
Thanks for catching the typo. Fixed it.